src/utils/versions.rs

```diff
@@ -43,9 +43,9 @@ impl CurrentVersions {
         let mut target = deployed.clone();
 
         if major_version {
-            target.increment_major();
```

Comment:
do we have tests for this? I think this doesn't work properly with the change

Reply:
I've moved away from using this util at all. Most things are passed in as command line arguments now